### PR TITLE
Add generated code and SDK API tests for Personalizer v1.0 preview

### PR DESCRIPTION
**Pull request by dwaijam**
Add generated code and SDK API tests for Personalizer v1.0 preview
Spec PR links
https://github.com/Azure/azure-rest-api-specs/pull/5743
https://github.com/Azure/azure-rest-api-specs/pull/5777
Both Spec PRs have been merged

### CODE REVIEW

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj
Comment (yangyuan) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj]: maybe a preview version as the service is public preview?

**Review by yangyuan — APPROVED** [review]

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj
Comment (dsgouda) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj]: @dwaijam Please mark the version as `0.8.0-preview`

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Properties/AssemblyInfo.cs
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Properties/AssemblyInfo.cs]: Please set this to `0.8.0.0`

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Properties/AssemblyInfo.cs
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Properties/AssemblyInfo.cs]: Please set this to `0.8.0.0`

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1]: Is the tag `release_1_0_preview` set to default in readme.md, if yes, this is not needed

Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1]: Please generate the code using this script once the Rest spec is merged. This should create a SdkInfo_*.cs and .txt files, please commit these files

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Comment by dsgouda** [comment]
Created for https://github.com/Azure/azure-rest-api-specs/pull/5743

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj
Comment (dwaijam) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj]: done

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Properties/AssemblyInfo.cs
Comment (dwaijam) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Properties/AssemblyInfo.cs]: done

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1
Comment (dwaijam) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1]: removed

Comment (dwaijam) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1]: added

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj]: I am not sure that is correct. Please check what REST API version this corresponds to

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj
Comment (dwaijam) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj]: fixed

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer.Tests/Personalizer.Tests.csproj
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer.Tests/Personalizer.Tests.csproj]: Test project version must be set to 1.0.0 and never modified. Not blocking but must be fixed

**Review by dsgouda — APPROVED** [review]
LGTM will merge once CIs pass

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer.Tests/Personalizer.Tests.csproj
Comment (dwaijam) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer.Tests/Personalizer.Tests.csproj]: fixed

**Pull request merged**
